[2023-12-11 19:14:15,766] INFO [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:5000
 * Running on http://192.168.15.158:5000 - call_trace=D:\Workspace\ML\rest_api\env\lib\site-packages\werkzeug\_internal.py L224 
[2023-12-11 19:14:15,767] INFO [33mPress CTRL+C to quit[0m - call_trace=D:\Workspace\ML\rest_api\env\lib\site-packages\werkzeug\_internal.py L224 
[2023-12-11 19:14:15,769] INFO  * Restarting with stat - call_trace=D:\Workspace\ML\rest_api\env\lib\site-packages\werkzeug\_internal.py L224 
[2023-12-11 19:14:18,567] ERROR Exception on /sleep [POST] - call_trace=D:\Workspace\ML\rest_api\env\lib\site-packages\flask\app.py L1449
Traceback (most recent call last):
  File "D:\Workspace\ML\rest_api\env\lib\site-packages\flask\app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "D:\Workspace\ML\rest_api\env\lib\site-packages\flask\app.py", line 1519, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "D:\Workspace\ML\rest_api\env\lib\site-packages\flask_cors\extension.py", line 165, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "D:\Workspace\ML\rest_api\env\lib\site-packages\flask\app.py", line 1517, in full_dispatch_request
    rv = self.dispatch_request()
  File "D:\Workspace\ML\rest_api\env\lib\site-packages\flask\app.py", line 1503, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "D:\Workspace\ML\rest_api\env\lib\site-packages\flask_openapi3\scaffold.py", line 191, in wrapper
    resp = _do_wrapper(
  File "D:\Workspace\ML\rest_api\env\lib\site-packages\flask_openapi3\do_wrapper.py", line 135, in _do_wrapper
    response = func(**request_kwargs)
  File "D:\Workspace\ML\rest_api\app.py", line 44, in login
    response = add_sleep(form)
  File "D:\Workspace\ML\rest_api\use_cases\add_sleep.py", line 14, in add_sleep
    Age = form.Age,
AttributeError: 'SleepSchema' object has no attribute 'Age'
[2023-12-11 19:14:18,605] INFO 127.0.0.1 - - [11/Dec/2023 19:14:18] "[35m[1mPOST /sleep HTTP/1.1[0m" 500 - - call_trace=D:\Workspace\ML\rest_api\env\lib\site-packages\werkzeug\_internal.py L224 
[2023-12-11 19:14:47,076] INFO  * Detected change in 'D:\\Workspace\\ML\\rest_api\\schemas\\sleep.py', reloading - call_trace=D:\Workspace\ML\rest_api\env\lib\site-packages\werkzeug\_internal.py L224 
[2023-12-11 19:14:47,249] INFO  * Restarting with stat - call_trace=D:\Workspace\ML\rest_api\env\lib\site-packages\werkzeug\_internal.py L224 
[2023-12-11 19:14:50,210] ERROR Exception on /sleep [POST] - call_trace=D:\Workspace\ML\rest_api\env\lib\site-packages\flask\app.py L1449
Traceback (most recent call last):
  File "D:\Workspace\ML\rest_api\env\lib\site-packages\flask\app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "D:\Workspace\ML\rest_api\env\lib\site-packages\flask\app.py", line 1519, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "D:\Workspace\ML\rest_api\env\lib\site-packages\flask_cors\extension.py", line 165, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "D:\Workspace\ML\rest_api\env\lib\site-packages\flask\app.py", line 1517, in full_dispatch_request
    rv = self.dispatch_request()
  File "D:\Workspace\ML\rest_api\env\lib\site-packages\flask\app.py", line 1503, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "D:\Workspace\ML\rest_api\env\lib\site-packages\flask_openapi3\scaffold.py", line 191, in wrapper
    resp = _do_wrapper(
  File "D:\Workspace\ML\rest_api\env\lib\site-packages\flask_openapi3\do_wrapper.py", line 135, in _do_wrapper
    response = func(**request_kwargs)
  File "D:\Workspace\ML\rest_api\app.py", line 44, in login
    response = add_sleep(form)
  File "D:\Workspace\ML\rest_api\use_cases\add_sleep.py", line 15, in add_sleep
    Sleep_duration = form.Sleep_duration,
AttributeError: 'SleepSchema' object has no attribute 'Sleep_duration'
[2023-12-11 19:14:50,214] INFO 127.0.0.1 - - [11/Dec/2023 19:14:50] "[35m[1mPOST /sleep HTTP/1.1[0m" 500 - - call_trace=D:\Workspace\ML\rest_api\env\lib\site-packages\werkzeug\_internal.py L224 
[2023-12-11 19:15:46,257] INFO  * Detected change in 'D:\\Workspace\\ML\\rest_api\\schemas\\sleep.py', reloading - call_trace=D:\Workspace\ML\rest_api\env\lib\site-packages\werkzeug\_internal.py L224 
[2023-12-11 19:15:46,450] INFO  * Restarting with stat - call_trace=D:\Workspace\ML\rest_api\env\lib\site-packages\werkzeug\_internal.py L224 
[2023-12-11 19:15:57,020] INFO  * Detected change in 'D:\\Workspace\\ML\\rest_api\\schemas\\sleep.py', reloading - call_trace=D:\Workspace\ML\rest_api\env\lib\site-packages\werkzeug\_internal.py L224 
[2023-12-11 19:15:57,082] INFO  * Restarting with stat - call_trace=D:\Workspace\ML\rest_api\env\lib\site-packages\werkzeug\_internal.py L224 
[2023-12-11 19:16:43,370] INFO  * Detected change in 'D:\\Workspace\\ML\\rest_api\\schemas\\sleep.py', reloading - call_trace=D:\Workspace\ML\rest_api\env\lib\site-packages\werkzeug\_internal.py L224 
[2023-12-11 19:16:43,444] INFO  * Restarting with stat - call_trace=D:\Workspace\ML\rest_api\env\lib\site-packages\werkzeug\_internal.py L224 
[2023-12-11 19:16:46,820] INFO  * Detected change in 'D:\\Workspace\\ML\\rest_api\\schemas\\sleep.py', reloading - call_trace=D:\Workspace\ML\rest_api\env\lib\site-packages\werkzeug\_internal.py L224 
[2023-12-11 19:16:46,883] INFO  * Restarting with stat - call_trace=D:\Workspace\ML\rest_api\env\lib\site-packages\werkzeug\_internal.py L224 
[2023-12-11 19:16:50,538] ERROR Exception on /sleep [POST] - call_trace=D:\Workspace\ML\rest_api\env\lib\site-packages\flask\app.py L1449
Traceback (most recent call last):
  File "D:\Workspace\ML\rest_api\use_cases\add_sleep.py", line 31, in add_sleep
    saidas = predictor.preditor(sleep.to_dict())
  File "D:\Workspace\ML\rest_api\models\ml_predictor.py", line 59, in preditor
    X_input = X_input.astype(float)
ValueError: could not convert string to float: 'undefined'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\Workspace\ML\rest_api\env\lib\site-packages\flask\app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "D:\Workspace\ML\rest_api\env\lib\site-packages\flask\app.py", line 1519, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "D:\Workspace\ML\rest_api\env\lib\site-packages\flask_cors\extension.py", line 165, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "D:\Workspace\ML\rest_api\env\lib\site-packages\flask\app.py", line 1517, in full_dispatch_request
    rv = self.dispatch_request()
  File "D:\Workspace\ML\rest_api\env\lib\site-packages\flask\app.py", line 1503, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "D:\Workspace\ML\rest_api\env\lib\site-packages\flask_openapi3\scaffold.py", line 191, in wrapper
    resp = _do_wrapper(
  File "D:\Workspace\ML\rest_api\env\lib\site-packages\flask_openapi3\do_wrapper.py", line 135, in _do_wrapper
    response = func(**request_kwargs)
  File "D:\Workspace\ML\rest_api\app.py", line 44, in login
    response = add_sleep(form)
  File "D:\Workspace\ML\rest_api\use_cases\add_sleep.py", line 44, in add_sleep
    logger.warning(f"Erro ao adicionar produto '{sleep.age}', {error_msg}")
AttributeError: 'Sleep' object has no attribute 'age'
[2023-12-11 19:16:50,543] INFO 127.0.0.1 - - [11/Dec/2023 19:16:50] "[35m[1mPOST /sleep HTTP/1.1[0m" 500 - - call_trace=D:\Workspace\ML\rest_api\env\lib\site-packages\werkzeug\_internal.py L224 
[2023-12-11 19:17:20,321] INFO  * Detected change in 'D:\\Workspace\\ML\\rest_api\\schemas\\sleep.py', reloading - call_trace=D:\Workspace\ML\rest_api\env\lib\site-packages\werkzeug\_internal.py L224 
[2023-12-11 19:17:20,510] INFO  * Restarting with stat - call_trace=D:\Workspace\ML\rest_api\env\lib\site-packages\werkzeug\_internal.py L224 
[2023-12-11 19:17:21,917] INFO  * Detected change in 'D:\\Workspace\\ML\\rest_api\\schemas\\sleep.py', reloading - call_trace=D:\Workspace\ML\rest_api\env\lib\site-packages\werkzeug\_internal.py L224 
[2023-12-11 19:17:22,109] INFO  * Restarting with stat - call_trace=D:\Workspace\ML\rest_api\env\lib\site-packages\werkzeug\_internal.py L224 
[2023-12-11 19:17:33,150] INFO  * Detected change in 'D:\\Workspace\\ML\\rest_api\\models\\sleep.py', reloading - call_trace=D:\Workspace\ML\rest_api\env\lib\site-packages\werkzeug\_internal.py L224 
[2023-12-11 19:17:33,341] INFO  * Restarting with stat - call_trace=D:\Workspace\ML\rest_api\env\lib\site-packages\werkzeug\_internal.py L224 
[2023-12-11 19:17:34,798] INFO  * Detected change in 'D:\\Workspace\\ML\\rest_api\\schemas\\sleep.py', reloading - call_trace=D:\Workspace\ML\rest_api\env\lib\site-packages\werkzeug\_internal.py L224 
[2023-12-11 19:17:34,990] INFO  * Restarting with stat - call_trace=D:\Workspace\ML\rest_api\env\lib\site-packages\werkzeug\_internal.py L224 
[2023-12-11 19:17:40,630] INFO  * Detected change in 'D:\\Workspace\\ML\\rest_api\\schemas\\sleep.py', reloading - call_trace=D:\Workspace\ML\rest_api\env\lib\site-packages\werkzeug\_internal.py L224 
[2023-12-11 19:17:40,822] INFO  * Restarting with stat - call_trace=D:\Workspace\ML\rest_api\env\lib\site-packages\werkzeug\_internal.py L224 
[2023-12-11 19:17:42,474] INFO 127.0.0.1 - - [11/Dec/2023 19:17:42] "[35m[1mPOST /sleep HTTP/1.1[0m" 500 - - call_trace=D:\Workspace\ML\rest_api\env\lib\site-packages\werkzeug\_internal.py L224 
